core/dbt/events/types.py

```diff
@@ -2499,8 +2491,8 @@ def message(self) -> str:
     SQLQueryStatus(status="", elapsed=0.1)
     SQLCommit(conn_name="")
     ColTypeChange(orig_type="", new_type="", table="")
-    SchemaCreation(relation=BaseRelation())
-    SchemaDrop(relation=BaseRelation())
+    SchemaCreation(relation=_make_key(BaseRelation()))
```

Comment:
I would have expected these lines to be: SchemaCreating(relation=_ReferenceKey). Is there some reason to call the _make_key function here?

Reply:
it's just shorter. this code is never run, just evaluated by mypy. happy to change it though if you think it's better.

Reply:
It's fine for now :)